pages/App.js

```diff
@@ -13,18 +16,69 @@ export default function List() {
     },
     {
       id: nanoid(),
-      text: "Hafermilch und Obst kaufen",
+      text: "die Welt retten",
+    },
+    {
+      id: nanoid(),
+      text: "Blumen gießen",
     },
   ]);
 
+  function addItem(event) {
+    event.preventDefault();
+    const inputElement = event.target.elements.newItem;
```

Comment:
Instead of reading the input you could use `FormData`.

```js
const formData = new FormData(event.target);
const formValues = Object.fromEntries(formData);
const inputValue = formValues.newItem;
```